lib/ts/types.ts

```diff
@@ -91,14 +91,21 @@ export function isValidFrontend(frontend: string): frontend is SupportedFrontend
 }
 
 type PythonFrameworks = "python-flask" | "python-drf" | "python-fastapi";
+type NodeJSFrameworks = "koa" | "express" | "nest";
 
-export type SupportedBackends = "node" | "nest" | "python" | PythonFrameworks | "go-http";
+export type SupportedBackends = "nodejs" | "python" | NodeJSFrameworks | PythonFrameworks | "go-http";
```

Comment:
please revert change of nodejs back to node

Reply:
Done